### PR TITLE
Microseconds Everywhere - Couchexport

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by czue** [comment]
build failing

File: setup.py [hunk 1]
```diff
@@ -12,13 +12,15 @@
...
         "openpyxl",
         "unidecode",
         "xlwt",
     ],
-    packages = find_packages(exclude=['*.pyc']),
-    include_package_data=True
+    packages=find_packages(exclude=['*.pyc']),
+    include_package_data=True,
+    dependency_links=[
+        'git+git://github.com/dimagi/dimagi-utils.git@2533a1aa96fd91a855b153e7bb9c15275ef844e7#egg=dimagi-utils-1.3usec',
```
Comment (czue) [inline setup.py]: fun - we should probably bump the version once https://github.com/dimagi/dimagi-utils/pull/266 is merged

**Comment by czue** [comment]
:white_check_mark: to merge

File: setup.py (hunk 1 above)
Comment (dannyroberts) [inline setup.py]: :+1:

**Comment by dannyroberts** [comment]
@czue good call on the auto-date format, I'd forgotten that we did that. Latest commit changes it to covert both input formats

**Comment by czue** [comment]
nice

**Pull request merged**